stdlib/public/Reflection/TypeLowering.cpp

```diff
@@ -607,7 +607,7 @@ const TypeRef *TypeConverter::getUnknownObjectTypeRef() {
   if (UnknownObjectTR != nullptr)
     return UnknownObjectTR;
 
-  UnknownObjectTR = BuiltinTypeRef::create(Builder, "BO");
+  UnknownObjectTR = BuiltinTypeRef::create(Builder, "yXl");
```

Comment:
Ok, keep this as "BO" -- the mangled name here doesn't 'mean' anything, it's just used to find the descriptor emitted in IRGen/GenReflection